src/gl/contourFeature.js

```diff
@@ -0,0 +1,305 @@
+//////////////////////////////////////////////////////////////////////////////
+/**
+ * Create a new instance of contourFeature
+ *
+ * @class
+ * @extends geo.contourFeature
+ * @returns {geo.gl.contourFeature}
+ */
+//////////////////////////////////////////////////////////////////////////////
+geo.gl.contourFeature = function (arg) {
+  'use strict';
+
+  if (!(this instanceof geo.gl.contourFeature)) {
+    return new geo.gl.contourFeature(arg);
+  }
+  arg = arg || {};
+  geo.contourFeature.call(this, arg);
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * @private
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  var m_this = this,
+      s_exit = this._exit,
+      m_textureUnit = 7,
```

Comment:
No particular reason, except that unit 0 is used in other features.

+1 :+1: absolutely.